heartbeat/Makefile.am

```diff
@@ -218,6 +218,7 @@ ocfcommon_DATA		= ocf-shellfuncs 	\
 			  ocf-rarun		\
 			  ocf-distro		\
 			  apache-conf.sh 	\
+			  aws.sh            \
```

Comment:
You should use tabs after aws.sh as well, so `\` lines up with the other lines.

You can run `git diff` for a preview of how it looks outside your editor.